test/controllers/ConfirmationControllerSpec.scala

```diff
@@ -92,12 +93,18 @@ class ConfirmationControllerSpec extends SpecBaseWithApplication with CoreTestDa
         121.58,
         0.0,
         0.0,
-        regularPaymentWithFullPeriod(2000.00, 2000.00, fullPeriodWithPaymentDate("2020-03-01", "2020-03-31", "2020-03-20"))),
+        regularPaymentWithFullPeriod(2000.00, 2000.00, fullPeriodWithPaymentDate("2020-03-01", "2020-03-31", "2020-03-20")),
```

Comment:
Are these always big decimals by default?

Reply:
a RegularPaymentWithFullPeriod takes Amount(value: BigDecimal), this is a helper function which creates an instance taking care of wrapping the BigDecimals in Amount(), otherwise test bodies would get even bigger than they are with a lot of calls to create Amount()